accepted/PSR-6-cache-meta.md

```diff
@@ -334,3 +334,22 @@ if (! (
     ));
 }
 ```
+
+### 8.2 Type additions
+
+The 2.0 release of the `psr/cache` package includes scalar parameter types.  The 3.0 release of the package includes return types.  This structure leverages PHP 7.2 covariance support to allow for a gradual upgrade process, but requires PHP 8.0 for type compatibility.
```

Comment:
Double spaces?

```suggestion
The 2.0 release of the `psr/cache` package includes scalar parameter types. The 3.0 release of the package includes return types. This structure leverages PHP 7.2 covariance support to allow for a gradual upgrade process, but requires PHP 8.0 for type compatibility.
```

Reply:
Are valid and common English.  The single-space oppressors do not have history on their side.

Reply:
Alright. I just haven't encountered it in any news website, book or guide.

Reply:
On the web you never see it because HTML folds all whitespace to a single space, regardless of what's in the source.  It is the underhanded weapon of our oppressors.